src/kernel/chainparams.cpp

```diff
@@ -130,6 +130,8 @@ class CMainParams : public CChainParams {
         consensus.vDeployments[Consensus::DEPLOYMENT_CHECKTEMPLATEVERIFY] = SetupDeployment{.bip = 119, .bip_version = 0, .never = true};
         consensus.vDeployments[Consensus::DEPLOYMENT_ANYPREVOUT] = SetupDeployment{.bip = 118, .bip_version = 0, .never = true};
 
+        consensus.vDeployments[Consensus::DEPLOYMENT_LNHANCE] = SetupDeployment{.bip = 999, .bip_version = 0, .never = true};
```

Comment:
Can you submit the proposal specs via https://github.com/bitcoin-inquisition/binana/pulls so we can have some canonical numbers here? Hopefully converting to md isn't too much of a pain?

Reply:
Added PRs for BIN-2024-000[34].

Reply:
Oh, these lines need to be updated after #46, swapping `.bip=X, .bip_version=Y` out for `.year=2024, .number=3, .revision=0` or so.